packages/generator-superset/generators/plugin-chart/templates/README.erb

```diff
@@ -1,7 +1,7 @@
 ## @superset-ui/plugin-chart-<%= packageName %>
```

Comment:
For the time being, the resulting plugin is more or less best suited to use in `superset-ui` so I think we should leave it for the time being. We're still working on other features that will make plugins better suited for external use, and when those are ready, I think it would make sense to add another choice in the Yeoman generator process, and name the package accordingly.